src/fio/FioEngine.ts

```diff
@@ -2139,11 +2147,9 @@ export const parseAction = ({
           networkFee = '0'
         } else {
           networkFee = isRecipient ? `-${fioAmount}` : fioAmount
+          networkFee = '0'
```

Comment:
This looks like a typo

Reply:
Yeah bad merge conflict resolution, good catch